cmobase/background.adoc

```diff
@@ -408,6 +408,9 @@ periodic cancellation of a reservation on another hart._
 * cache block size for zero
 * CBIE support at each privilege level
 * what happens if CBO operations are executed on addresses that are not cached?
+* physical addresses may differ between observers (remapping mechanisms)
+* refine the concept of "uniquely identified by a physical address" in a system
```

Comment:
I'll add these to the open issues section.

However, I've been assuming that "physical address" is the canonical identifier for a given memory location. There may certainly be mappers between various agents that may introduce aliases from different perspectives, though there's not really an architectural concept for those.

Reply:
Indeed - this issue is not specific to the CMO specification; it's something that probably should be addressed across the ISA specifications, at some point.  Please feel free to remove it if you feel this isn't the right place to track it

Reply:
I've made a note in the spec so it doesn't get lost.